src/main/java/pl/isa/thebuggers/fitly/Diet.java

```diff
@@ -0,0 +1,163 @@
+//        double dietHeight = scanner.nextDouble();
+//        System.out.print("Enter your age (in years): ");
+//        int age = scanner.nextInt();
+//
+//        int activityLevelOption;
+//        String activityLevel = null;
+//        do {
+//            System.out.println("Specify your activity level:");
+//            System.out.println("1. Couch potato mode");
+//            System.out.println("2. Couch potato with occasional dance breaks");
+//            System.out.println("3. Parkour panda mode");
+//            System.out.println("4. Marathoner chasing ice cream truck");
+//            System.out.println("5. Superhero or ninja warrior level");
+//            System.out.print("Type your choice: ");
+//            try {
+//                activityLevelOption = scanner.nextInt();
+//                switch (activityLevelOption) {
+//                    case 1:
+//                        activityLevel = "Couch potato mode";
+//                        break;
+//                    case 2:
+//                        activityLevel = "Couch potato with occasional dance breaks";
+//                        break;
+//                    case 3:
+//                        activityLevel = "Parkour panda mode";
+//                        break;
+//                    case 4:
+//                        activityLevel = "Marathoner chasing ice cream truck";
+//                        break;
+//                    case 5:
+//                        activityLevel = "Superhero or ninja warrior level";
+//                        break;
+//                    default:
+//                        System.out.println("Invalid input. Please enter a number between 1 and 5.");
+//                        activityLevel = "Invalid";
+//                        break;
+//                }
+//            } catch (IllegalArgumentException e) {
+//                activityLevelOption = 0;
+//            }
+//        } while (activityLevelOption < 1 || activityLevelOption > 5);
+//
+//        System.out.println("Selected activity level: " + activityLevel);
+//
+//        double bmr;
+//        if (isMale()) {
+//            bmr = Diet.calculateBmrForMale(dietWeight, dietHeight, age);
+//        } else {
+//            bmr = Diet.calculateBmrForFemale(dietWeight, dietHeight, age);
+//        }
+//
+//        double activityFactor = Diet.getActivityFactor(activityLevel);
+//        double dailyCalorieNeeds = bmr * activityFactor;
+//        String selectedDietFile = Diet.selectAndDisplayDietFile(dailyCalorieNeeds);
+
+
+//      TODO:#2 add this to Main in Main.java for Case 1 to work correctly
+//    private static boolean isMale() {
+//        return true;
+//    }
+
+package pl.isa.thebuggers.fitly;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Diet {
+
+    private static final double maleBmrConstant = 88.362;
```

Comment:
Nazwy stałych piszemy raczej w taki sposób: MALE_BMR_CONSTANT, dzięki czemu wiadomo, że to stała, a nie zmienna